NickvisionTagger.Shared/Controllers/MainWindowController.cs

```diff
@@ -588,6 +599,48 @@ public void ExportSelectedAlbumArt(string path, AlbumArtType type)
         }
     }
 
+    /// <summary>
+    /// Adds a custom property to the selected music files
+    /// </summary>
+    /// <param name="name">The name of the property to add</param>
+    public void AddCustomProperty(string name)
+    {
+        var added = false;
+        foreach(var pair in SelectedMusicFiles)
+        {
+            pair.Value.SetCustomProperty(name, "");
+            MusicFileSaveStates[pair.Key] = false;
+            added = true;
+        }
+        if(added)
```

Comment:
If `added` is set to true on any iteration, why not just check if the length of SelectedMusicFiles > 0?